metadata-ws/src/test/resources/features/accession.feature

```diff
@@ -0,0 +1,68 @@
+Feature: accession object
+
+  Scenario Outline: find different objects by accession and version
+    When I request POST /taxonomies with JSON payload:
+    """
+    {
+      "taxonomyId": 9606,
+      "name": "Homo Sapiens"
+    }
+    """
+    Then set the URL to TEST_TAXONOMY
+    When I create a test parameterized study with EGAS0001 for accession, 1 for version, test_study for name, false for deprecated, 0 for releaseDay and TEST_TAXONOMY for taxonomy
+    And I request POST /files with JSON payload:
+    """
+    {
+      "accessionVersionId": {
+        "accession": "EGAF0001",
+        "version": 1
+        },
+      "hash": "asd123",
+      "name": "file1",
+      "size": 100,
+      "type": "TSV"
+    }
+    """
+    Then set the URL to TEST_FILE
+    When I request POST /taxonomies with JSON payload:
+    """
+    {
+      "taxonomyId": 1,
+      "name": "Species1"
+    }
+    """
+    Then set the URL to TEST_TAXONOMY1
+    When I request POST /taxonomies with JSON payload:
+    """
+    {
+      "taxonomyId": 2,
+      "name": "Species2"
+    }
+    """
+    Then set the URL to TEST_TAXONOMY2
```

Comment:
I don't see a reason why you have to create another two taxonomies (`TEST_TAXONOMY1` and `TEST_TAXONOMY2`) here. The taxonomies are not related to the test cases laid out in this scenario. Could you not re-use the other one (`TEST_TAXONOMY`) instead?

Reply:
https://github.com/EBIvariation/metadata-ws/pull/46#discussion_r272830455.
Separate taxonomy for studies and sample; otherwise one common and other exclusive